packages/ovh-manager/webpack.prod.js

```diff
@@ -1,4 +1,12 @@
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+
 module.exports = {
   mode: 'production',
   devtool: 'source-map',
+  plugins: [
+    new CleanWebpackPlugin(['dist'], {
```

Comment:
Shouldn't be in the `webpack.common.js` file instead?

I used to deal a lot with the [Yeoman Generator Web app](https://github.com/yeoman/generator-webapp) by the past and as you can see the `clean` task is used both on `default` and `serve` as well.

* [`serve` task](https://github.com/yeoman/generator-webapp/blob/master/app/templates/gulpfile.js#L110).
* [`default` task](https://github.com/yeoman/generator-webapp/blob/master/app/templates/gulpfile.js#L204).

Feel free to tell me what's the benefit in your case:)

Reply:
In development mode, we don't generate a dist folder @antleblanc